author: Andres Salomon <dilinger@debian.org>

Remove clang build flags that clang-18 doesn't support. Can be dropped
when we move to a newer version of clang.

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -1967,10 +1967,8 @@ config("default_warnings") {
         ]
 
         # TODO(crbug.com/344680447): Fix and re-enable.
-        cflags_cc += [ "-Wno-missing-template-arg-list-after-template-kw" ]
 
         # TODO(crbug.com/356172342): Fix and re-enable.
-        cflags_cc += [ "-Wno-dangling-assignment-gsl" ]
       }
     }
 
